MapleServer2/Packets/GemPacket.cs

```diff
@@ -10,7 +11,18 @@ public static class GemPacket
         private enum GemMode : byte
         {
             EquipItem = 0x00,
-            UnequipItem = 0x01
+            UnequipItem = 0x01,
+            EquipError = 0x04
+        }
+
+        public enum GemEquipError : short
+        {
+            InventoryFull,
+            SlotNoLongerValid,
+            ItemNotEligible,
+            ItemExpired,
+            ItemPCCafeOnly,
+            ActivePremiumClubRequired
```

Comment:
Any numerical values for these enums?

Reply:
They're in a linear array, long as they're ordered numbers are redundant. For readability it's possible to label them 0~5 though. Should I consider numbered Enums as a standard style guide for this project?

Reply:
Nah I was just curious if they had any values at all, I'm cool with them being unlabeled in that case